ci/tasks/test-gcs-blobstore-client-integration.yml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
That's the plan, getting the main pipeline setup to test is going to be an ordeal so I'm deferring for now.